test/parallel/test-fs-realpath.js

```diff
@@ -197,7 +197,7 @@ function test_cyclic_link_protection(callback) {
     fs.symlinkSync(t[1], t[0], 'dir');
     unlink.push(t[0]);
   });
-  assert.throws(function() { fs.realpathSync(entry); }, /ELOOP/);
+  assert.throws(function() { fs.realpathSync(entry); });
```

Comment:
@bzoz Ah, I see now – are you sure these changes were specific to the C realpath implementation? I know I requested this particular change here (adding `/ELOOP/`) because this test had also passed when an error other than `ELOOP` was thrown (thus masquerading a real bug); all implemetations should throw `ELOOP` here.

I’ll be trying it out myself in a second, but I think you can safely keep all test changes from Trevor’s commit.